suites/pacific/rgw/tier-2_rgw_ms_failover_failback.yaml

```diff
@@ -1,4 +1,4 @@
-# Test suite for evaluating RGW multi-site deployment scenario.
```

Comment:
@TejasC88 I think you deleted this file by mistake suites/reef/rgw/tier-1_rgw_multisite.yaml
you moved this file
suites/reef/rgw/tier-1_rgw_multisite.yaml → suites/pacific/rgw/tier-2_rgw_ms_failover_failback.yaml
please check..